CMakeLists.txt

```diff
@@ -0,0 +1,362 @@
+# Copyright 2017 The LEVELDB Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+cmake_minimum_required(VERSION 3.1)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/cmake/toolchain.cmake")
```

Comment:
Set it to cache so user can customize it:
* https://cgold.readthedocs.io/en/latest/tutorials/variables/cache.html#use-case